theano/gpuarray/dnn_base.c

```diff
@@ -88,6 +92,13 @@ c_set_filter(PyGpuArrayObject *var, cudnnFilterDescriptor_t desc) {
     dims[i] = PyGpuArray_DIM(var, i);
   }
 
+  /* Filters can't be less than 3d so we pad */
```

Comment:
I mean, add that at current cudnn v5, there is this limit. So if we end up in the futur where the current and the previous don't have this limit and we still see this code, we now easily we can remove it. Also, if we drop cudnn v5, it make the clean up faster and make sure we won't forget to check that.